adminSiteServer/testPageRouter.tsx

```diff
@@ -437,19 +438,23 @@ function EmbedTestPage(props: EmbedTestPageProps) {
     )
 }
 
-testPageRouter.get("/embeds", async (req, res) => {
-    const props = await db.knexReadonlyTransaction((trx) =>
-        propsFromQueryParams(trx, {
+getPlainRouteWithROTransaction(
```

Comment:
At http://localhost:3030/admin/test , only the explorer test embeds appear to work

Everything else gets:
```
TypeError: The operator "undefined" is not permitted
```

Reply:
Ah nice, thanks for catching that. Fixed on the new top stack PR #3391